torch/nn/modules/activation.py

```diff
@@ -789,10 +788,15 @@ def forward(self, query, key, value, key_padding_mask=None,
           the embedding dimension.
         - value: :math:`(S, N, E)` where S is the source sequence length, N is the batch size, E is
           the embedding dimension.
-        - key_padding_mask: :math:`(N, S)`, ByteTensor, where N is the batch size, S is the source sequence length.
+        - key_padding_mask: :math:`(N, S)` where N is the batch size, S is the source sequence length.
+          If a ByteTensor is provided, ``1`` is filled with ``-inf`` while ``0`` is filled with ``0.0``.
```

Comment:
I wouldn't describe how this is implemented, but just what it does for each dtype. So you can drip the "is filled with inf while 0 is filled with 0.0", but say "if 1 it's included, if 0 the value is ignored" or something like that.